access_seats.go

```diff
@@ -0,0 +1,64 @@
+package cloudflare
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/goccy/go-json"
+)
+
+var errMissingAccessSeatUID = errors.New("missing required access seat UID")
+
+type AccessUserSeat struct {
+	AccessSeat  bool   `json:"access_seat"`
+	CreatedAt   string `json:"created_at"`
+	GatewaySeat bool   `json:"gateway_seat"`
+	SeatUID     string `json:"seat_uid"`
+	UpdatedAt   string `json:"updated_at"`
+}
+
+// Removes a user from a Zero Trust seat when both access_seat and gateway_seat are set to false.
+type UpdateAccessUserSeatParams struct {
+	SeatUID     string `json:"seat_uid"`
+	AccessSeat  bool   `json:"access_seat"`
+	GatewaySeat bool   `json:"gateway_seat"`
+}
+
+type UpdateAccessUserSeatResponse struct {
+	Success    bool           `json:"success"`
+	Errors     []string       `json:"errors"`
+	Messages   []string       `json:"messages"`
+	Result     AccessUserSeat `json:"result"`
+	ResultInfo ResultInfo     `json:"result_info"`
+}
+
+// UpdateAccessUserSeat updates a Access User Seat.
+// UpdateAccessUserSeat Removes a user from a Zero Trust seat when both access_seat and gateway_seat are set to false.
+//
+// Access API reference: https://developers.cloudflare.com/api/operations/zero-trust-seats-update-a-user-seat
```

Comment:
Mirrored the comment structure of `account_roles.go` but happy to structure this as needed.